hdf5.spec

```diff
@@ -30,16 +29,14 @@ Source0: https://github.com/HDFGroup/%{name}/archive/%{name}-%{hdf5_tag}.tar.gz
 Source1: h5comp
 # For man pages
 Source2: http://ftp.us.debian.org/debian/pool/main/h/hdf5/hdf5_1.12.0+repack-1~exp2.debian.tar.xz
-Patch0: hdf5-shared-lib.patch
```

Comment:
This looks like a local patch file that should be removed from the source tree if it's not being used any more.